chromium_src/third_party/blink/renderer/modules/webgl/webgl2_rendering_context_base.cc

```diff
@@ -43,7 +43,7 @@ ScriptValue FarbleGLInt64Parameter(WebGL2RenderingContextBase* owner,
   if (!owner->isContextLost())
     owner->ContextGL()->GetInteger64v(pname, &value);
   if (value > 0) {
-    std::mt19937_64 prng =
+    brave::FarblingPRNG prng =
```

Comment:
I wonder how it appears here? `components/brave_shields/browser` should not be linked into `blink`. only `common` should be shared between browser/renderer codebase.

Reply:
oh, I see it in `chromium_src/third_party/blink/renderer/core/execution_context/execution_context.h`, but also in other parts of the code.
If you can create a single file that declares `FarblingPRNG` and somehow include it everywhere (with proper gn deps), then it would be nice.

Reply:
@goodov I don't think blink can have a dep on components

Reply:
> @goodov I don't think blink can have a dep on components

it can. you can see it in many places across `blink`:
https://source.chromium.org/search?q=%2F%2Fcomponents%20file:.gn&ss=chromium%2Fchromium%2Fsrc:third_party%2Fblink%2F

I don't like the idea of copy-pasting typedefs, we can do better.